app/src/main/java/com/rahobbs/todo/helpers/Feedback.java

```diff
@@ -0,0 +1,23 @@
+package com.rahobbs.todo.helpers;
+import android.content.Intent;
+import android.util.Log;
+
+import com.rahobbs.todo.BuildConfig;
+
+/**
+ * Created by rachael on 2/22/16.
+ * Helper class for Send Feedback button
+ */
+public class Feedback {
+
+
+    public Intent sendFeedback(Intent i) {
```

Comment:
make this static so when you need this you don't have to instantiate the class you can just call `Feedback.sendFeedback()`

Also I would consider renaming the class and maybe the method to something more general